packages/rancher-logging/overlay/templates/loggings/rke2/logging-rke2-containers.yaml

```diff
@@ -21,6 +21,12 @@ spec:
     tolerations:
       {{ toYaml . | nindent 6 }}
     {{ end }}
+      - key: "OS"
```

Comment:
Does fluentbit work on windows?

Reply:
Appears to be only on a case by case basis with certain Git tags.

https://github.com/fluent/fluent-bit/issues/960
https://github.com/fluent/fluent-bit/issues/2351
https://github.com/fluent/fluent-bit/tags